python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -41,6 +43,10 @@
     "UpdateInstructions",
     ["node_id", "init_commands", "start_ray_commands", "docker_config"])
 
+AutoscalerSummary = namedtuple(
+    "AutoscalerSummary",
+    ["active_nodes", "pending_nodes", "pending_launches", "failed_nodes"])
```

Comment:
Can we print "starting nodes" or "launching nodes" instead of pending nodes?

Reply:
I prefer to leave it as pending since that's what was in the RFC